src/plugins/intel_cpu/tests/functional/single_layer_tests/instances/arm/eltwise.cpp

```diff
@@ -0,0 +1,59 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "shared_test_classes/single_layer/mvn.hpp"
+#include "common_test_utils/test_constants.hpp"
+
+using namespace LayerTestsDefinitions;
+
+namespace CPULayerTestsDefinitions {
+namespace Eltwise {
+
+std::vector<InferenceEngine::Precision> dataPrecisions = {
+        InferenceEngine::Precision::FP16,
+        InferenceEngine::Precision::FP32
+};
+
+std::vector<InferenceEngine::Precision> idxPrecisions = {
+        InferenceEngine::Precision::I32,
+        InferenceEngine::Precision::I64
+};
+
+const std::vector<float> epsilonF = {
+        0.0001f
+};
+
+const std::vector<std::string> epsMode = {
+        "inside_sqrt",
+        "outside_sqrt"
+};
+
+INSTANTIATE_TEST_SUITE_P(smoke_power_mvn_decomposition_3D, Mvn6LayerTest,
```

Comment:
Mvn6LayerTest is seemed to be the class from the shared_tests.
Here we should work only with the plugin's single_layer_tests
Also, I do not think we need to deal with MVN here at all.
I understand that this eltwise-power node is the result of MVN decomposition.
But for accuracy verification we can just create an instance for the eltwise-power test (even the file itself is called eltwise.cpp)

Reply:
@EgorDuplensky done